services/alertmanager/alertmanager_test.go

```diff
@@ -36,3 +46,123 @@ func TestAlertmanager(t *testing.T) {
 
 	require.NoError(t, svc.IsReady(context.Background()))
 }
+
+func TestPopulateConfig(t *testing.T) {
+	var cfg alertmanager.Config
+	buf, err := ioutil.ReadFile(alertmanagerBaseConfigPath)
+	require.NoError(t, err)
+	err = yaml.Unmarshal(buf, &cfg)
+	require.NoError(t, err)
+
+	sqlDB := testdb.Open(t, models.SkipFixtures, nil)
+	db := reform.NewDB(sqlDB, postgresql.Dialect, reform.NewPrintfLogger(t.Logf))
+	tx, err := db.Begin()
+	require.NoError(t, err)
+	defer func() {
+		require.NoError(t, tx.Rollback())
+	}()
+
+	q := tx.Querier
+
+	channel1, err := models.CreateChannel(q, &models.CreateChannelParams{
```

Comment:
This channel is created in one transaction. See the other comment below.

The simplest solution is to use db.Querier there and not create test transaction